tests/unittest_data/decision_forest_regression_batch.csv

```diff
@@ -1,127 +1,127 @@
-36.70242652

```

Comment:
What is the need for changing data file?

Reply:
See the PR description. Node splitting will be changed with https://github.com/oneapi-src/oneDAL/pull/2292, resulting in different values. As far as I can tell, the algorithm is still producing correct results.